### ISSUE TITLE
Should the PDF-hul log offsets wherever possible

### ISSUE THREAD

**Issue by carlwilson** [openpreserve/jhove#947]
One is for @asciim0's input, and I'd also be curious to know @samalloing's thoughts. The PDF-hul could be more consistent regarding whether it logs an error's offset. There are multiple places where offsets could be added to give better information. The downside is that adding this info might lead to multiple error reporting in some circumstances. The extra information may prevent some errors from getting collated into a single instance, as they are no longer considered identical.

**Comment by samalloing** [comment]
Hi @carlwilson 
I think it is fine as the offset is in an attribute. But I think these two errors (PDF-HUL-1 and 149) can be changed in the future to describe the errors better. But that's another pull request ;-)

Sam

**Comment by asciim0** [comment]
hey @carlwilson ! as mentioned on our call today i think for PDF it would be much more helpful if jhove could return the obj id instead of (or in addition to) the offset.

**Comment by carlwilson** [comment]
@asciim0 I assume you mean #647

**Comment by asciim0** [comment]
yes! apparently i've requested it all over the board throughout the years :-D
